assets/panel/debugger.properties

```diff
@@ -601,7 +601,11 @@ sourceFooter.codeCoverage=Code coverage
 
 # LOCALIZATION NOTE (sourceFooter.currentCursorPosition): Text associated
 # with the current cursor line and column
-sourceFooter.currentCursorPosition=(%S, %S)
+sourceFooter.currentCursorPosition.label=(%1$S, %2$S)
```

Comment:
No need to update the ID here, localizers can already change it if they want to.